src/batInt.mli

```diff
@@ -387,4 +387,38 @@ module Safe_int : sig
   (** Equality function for integers, useful for {!HashedType}. *)
 
   val ord : t -> t -> BatOrd.order
+
+end
+
+module Set :
+sig
+  type elt = int
+  type t
+  val empty : t
+  val is_empty : t -> bool
+  val mem : elt -> t -> bool
+  val add : elt -> t -> t
+  val singleton : elt -> t
+  val remove : elt -> t -> t
+  val union : t -> t -> t
+  val inter : t -> t -> t
+  val diff : t -> t -> t
+  val compare : t -> t -> elt
+  val equal : t -> t -> bool
+  val subset : t -> t -> bool
+  val iter : (elt -> unit) -> t -> unit
+  val fold : (elt -> 'a -> 'a) -> t -> 'a -> 'a
+  val for_all : (elt -> bool) -> t -> bool
+  val exists : (elt -> bool) -> t -> bool
+  val filter : (elt -> bool) -> t -> t
+  val partition : (elt -> bool) -> t -> t * t
+  val cardinal : t -> elt
+  val elements : t -> elt list
+  val min_elt : t -> elt
+  val max_elt : t -> elt
+  val choose : t -> elt
+  val split : elt -> t -> t * bool * t
+  val find : elt -> t -> elt
+  val of_list : elt list -> t
 end
+(** Ready to use intset. *)
```

Comment:
Yeah, but this isn't `Set.Make` -- it's `Set.S`, which already has an appropriate docstring (it will be shown in the output of both ocamldoc and ocp-index).

Perhaps a better comment would be "A set of integers"?